specs/Deployment/DeploymentAPI.md

```diff
@@ -221,6 +266,26 @@ namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime
         /// WindowsAppSDK main and singleton packages will be shut down forcibly if they are
         /// currently in use, when registering the WinAppSDK packages.
         Boolean ForceDeployment;
+
+        /// If not successful call DebugBreak()
+        [contract(DeploymentContract, 3)]
+        Boolean OnError_DebugBreak;
+
+        /// If not successful call DebugBreak() if a debugger is attached to the process
+        [contract(DeploymentContract, 3)]
+        Boolean OnError_DebugBreak_IfDebuggerAttached;
+
+        /// If not successful perform a fail-fast
+        [contract(DeploymentContract, 3)]
+        Boolean OnError_FailFast;
+
+        /// If not successful show UI
```

Comment:
The order of the options in DeploymentInitializeOptions definition doesn't match the order they're applied in the examples section.  I assume the order of application is deterministic.  This should be clear in the API details or comments in this section.

Reply:
The behavior will be doc'd (needs to be). Reordering physical order of the properties isn't a substitute for that, is it?

Reply:
DISCUSSION: Only 1 option now so this comment falls out / NA